### PR TITLE
chore: [MICROBA-1681] update readme, fix badges

### PR DESCRIPTION
**Pull request by Tj-Tracy**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -22,30 +27,41 @@ For now, this repo is not intergrated with devstack. You'll be running the app l
...
+      git clone https://github.com/edx/frontend-app-communications.git
 
 2. Install frontend dependencies
 
    .. code-block::
 
       npm i
 
-3. start the devserver. The app will be running at `localhost:8585`, or whatever port you change it too.
+3. start the devserver. The app will be running at ``localhost:1984``, or whatever port you change it too.
```
Comment (hurtstotouchfire) [inline README.rst:38]: where does this magic number come from?

Comment (Tj-Tracy) [reply README.rst:38]: I made it up! it lives here: https://github.com/edx/frontend-app-communications/blob/8d1cc534b7b3a53841e417b2e4441eb4bc20031d/.env.development#L2

File: README.rst [hunk 2]
```diff
@@ -22,30 +27,41 @@ For now, this repo is not intergrated with devstack. You'll be running the app l
...
 
    .. code-block::
 
-      npm i
+      npm start
```
Comment (justinhynes) [inline README.rst:42]: Dumb question -- do we not have to do the `npm i` the first time?

File: README.rst [hunk 3]
```diff
@@ -22,30 +27,41 @@ For now, this repo is not intergrated with devstack. You'll be running the app l
...
+1. In ``.env.development``
+2. Added to the ``mergeConfig`` found in ``src/index.jsx``
+
+.. code-block::
+
+   initialize({
+      config: () => {
+       mergeConfig({
+       EXAMPLE_VALUE: true,
+       }, 'CommuncationsAppConfig');
```
Comment (justinhynes) [inline README.rst:58]: Should we add this to the cookie cutter app Readme? It seems like good advice for any MFE that is created.

File: README.rst [hunk 4]
```diff
@@ -22,30 +27,41 @@ For now, this repo is not intergrated with devstack. You'll be running the app l
...
 
 Running Tests
 ---------------------------
 
-tests use jest as usual. To run all the tests for this repo:
+tests use `jest` and `react-test-library`. To run all the tests for this repo:
```
Comment (justinhynes) [inline README.rst:63]: Nit: Sentence begins with a lowercase letter.

File: README.rst [hunk 5]
```diff
@@ -22,30 +27,41 @@ For now, this repo is not intergrated with devstack. You'll be running the app l
...
    .. code-block::
 
-      npm i
+      npm start
 
 Environment Variables/Setup Notes
 ---------------------------------
 
-TBD
+if you wish to add new environment varibles for local testing, they should be listed in 2 places:
```
Comment (justinhynes) [inline README.rst:47]: Nit: Sentence begins with a lowercase letter.

File: README.rst (hunk 1 above)
Comment (justinhynes) [inline README.rst:38]: Nit: Sentence begins with a lowercase letter.

**Review by justinhynes — APPROVED** [review]

File: README.rst (hunk 2 above)
Comment (Tj-Tracy) [reply README.rst:42]: you do, that step is listed above this one. for some reason, both the install and starting the dev server had `npm i` as the command

File: README.rst (hunk 3 above)
Comment (Tj-Tracy) [reply README.rst:58]: This information is listed else-where in the documentation for the cookie cutter. I just put it here for more vision into something we do fairly often.

File: README.rst (hunk 2 above)
Comment (Tj-Tracy) [reply README.rst:42]: you do, that step is listed above this one. for some reason, both the install and starting the dev server had `npm i` as the command

File: README.rst (hunk 3 above)
Comment (Tj-Tracy) [reply README.rst:58]: This information is listed else-where in the documentation for the cookie cutter. I just put it here for more vision into something we do fairly often.

File: README.rst (hunk 2 above)
Comment (justinhynes) [reply README.rst:42]: Oh, I missed that! Woops. Thanks :)

**Pull request merged**